certif/tests/integration/components/issue-report-modal/non-blocking-candidate-issue-certification-issue-report-fields_test.js

```diff
@@ -34,16 +34,19 @@ module(
     test('it should show textarea if category is checked', async function (assert) {
```

Comment:
quid d'un test qui s'assure que ça ferme la modale et que le signalement s'affiche ? (cas passant)

Reply:
Peut-être rajouté dans une autre PR afin de ne pas bloquer la MEP ?

Reply:
Déjà testé dans `session-finalization_test.js` `it should show "Ajouter / Supprimer" button`

Reply:
Ouais mais on ne remplissait pas un textarea dans ce test